xbmc/CueDocument.cpp

```diff
@@ -75,10 +75,7 @@ class FileReader
   : public CueReader
 {
 public:
-  explicit FileReader(const std::string &strFile) : m_szBuffer{}
-  {
-    m_opened = m_file.Open(strFile);
-  }
+  explicit FileReader(const std::string& strFile) { m_opened = m_file.Open(strFile); }
```

Comment:
one question, better to init `m_opened` in the member initializer list?
```cpp
explicit FileReader(const std::string& strFile) : m_opened(m_file.Open(strFile)) {}
```

Reply:
In general member initializer list is preferred, but with the current enabled checks of `cppcheck` and `clang-tidy` it's not reported. This PR only fixes warnings reported by those tools.